src/OrbitGl/PrimitiveAssemblerTest.cpp

```diff
@@ -154,28 +156,31 @@ TEST(PrimitiveAssembler, ComplexShapes) {
 
   primitive_assembler_tester.StartNewFrame();
 
-  // Draw arrow pointing down
-  primitive_assembler_tester.AddVerticalArrow(kBottomRight, kArrowSize, 0, kFakeColor,
-                                              PrimitiveAssembler::ArrowDirection::kDown,
-                                              kArrowHeadSize);
+  // AddVerticalArrow (arrow pointing down) -> 1 box and 1 Triangle
+  primitive_assembler_tester.AddVerticalArrow(kArrowStartingPos, kArrowBodySize, kArrowHeadSize, 0,
+                                              kFakeColor,
+                                              PrimitiveAssembler::ArrowDirection::kDown);
+  float arrow_width1 = std::max(kArrowHeadSize[0], kArrowBodySize[0]);
+  float arrow_height1 = kArrowHeadSize[1] + kArrowBodySize[1];
   EXPECT_EQ(primitive_assembler_tester.GetNumTriangles(), 1);
   EXPECT_EQ(primitive_assembler_tester.GetNumLines(), 0);
   EXPECT_EQ(primitive_assembler_tester.GetNumBoxes(), 1);
   EXPECT_TRUE(primitive_assembler_tester.IsEverythingInsideRectangle(
-      kBottomRight - Vec2{kArrowHeadSize[0], 0}, Vec2{2 * kArrowHeadSize[0], kArrowSize[1]}));
+      kArrowStartingPos - Vec2{arrow_width1 / 2, 0}, Vec2{arrow_width1, arrow_height1}));
 
   primitive_assembler_tester.StartNewFrame();
 
-  // Draw arrow pointing up
-  primitive_assembler_tester.AddVerticalArrow(kBottomRight, kArrowSize, 0, kFakeColor,
-                                              PrimitiveAssembler::ArrowDirection::kUp,
-                                              kArrowHeadSize);
+  // AddVerticalArrow (arrow pointing up) -> 1 box and 1 Triangle
+  primitive_assembler_tester.AddVerticalArrow(kArrowStartingPos, kArrowBodySize, kArrowHeadSize, 0,
+                                              kFakeColor, PrimitiveAssembler::ArrowDirection::kUp);
+  float arrow_width2 = std::max(kArrowHeadSize[0], kArrowBodySize[0]);
```

Comment:
Nit: I didn't need/consider that before but you can just use arrow_width in both places by scoping each part.

```
{
   // AddVerticalArrow (arrow pointing down) -> 1 box and 1 Triangle
   float arrow_width = ....
}

{
   // AddVerticalArrow (arrow pointing up) -> 1 box and 1 Triangle
   float arrow_width = ....
}

```